pkgs/by-name/de/deepfilternet/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "deepfilternet";
+  version = "0.5.6";
+
+  src = fetchFromGitHub {
+    owner = "Rikorose";
+    repo = "DeepFilterNet";
+    rev = "v${version}";
+    hash = "sha256-5bYbfO1kmduNm9YV5niaaPvRIDRmPt4QOX7eKpK+sWY=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "hdf5-0.8.1" = "sha256-qWF2mURVblSLPbt4oZSVxIxI/RO3ZNcZdwCdaOTACYs=";
+    };
+  };
+
+  # only the ladspa plugin part has been packaged so far...
+
+  buildAndTestSubdir = "ladspa";
+
+  postInstall = ''
+    mkdir $out/lib/ladspa
+    mv $out/lib/libdeep_filter_ladspa.so $out/lib/ladspa/libdeep_filter_ladspa.so
```

Comment:
If replacing .so with the stdenv.hostPlatform.extensions.sharedLibrary variable doesn't fix Darwin, we could just mark it broken.